### PR TITLE
Update Woo Purple

### PR DESCRIPTION
**Pull request by brentmackinnon**
Updating the backgrounds, logos, and favicons so we're using the right Woo Purple in the slate presentation of the docs.

### CODE REVIEW

File: source/stylesheets/_variables.scss [hunk 1]
```diff
@@ -28,18 +28,18 @@ $examples-bg: #393939 !default;
...
 $lang-select-bg: #222 !default;
 $lang-select-active-bg: $examples-bg !default; // feel free to change this to blue or something
 $lang-select-pressed-bg: #111 !default; // color of language tab bg when mouse is pressed
 $main-bg: #ffffff !default;
-$aside-notice-bg: #9B5C8F !default;
+$aside-notice-bg: #7f54b3 !default;
 $aside-warning-bg: #ca4949 !default;
 $aside-success-bg: #38a845 !default;
 $search-notice-bg: #c97a7e !default;
-$link-color: #804877;
```
Comment (jonathansadowski) [inline source/stylesheets/_variables.scss:42]: We were previously using a different purple here than the bg colors. Do you know if this should be the case here as well?

**Review by jonathansadowski — APPROVED** [review]
I just added a question about the link color, which was previously distinct from the background colors... it might be the case that it not being distinct is okay, but wanted to double-check first. I'll approve this, and if it's not an oversight, feel free to merge. Thanks for cleaning this up.

File: source/stylesheets/_variables.scss (hunk 1 above)
Comment (brentmackinnon) [reply source/stylesheets/_variables.scss:42]: @jonathansadowski good question. I noticed that it was quite similar, and I wasn't sure the reason for having two similar colours, so I used our Woo purple for both here. 

old link color: IMG002397 
new link color: IMG002399


old bg color: IMG002398 
new bg color:IMG002399

Comment (jonathansadowski) [reply source/stylesheets/_variables.scss:42]: Okay, yep, that seems reasonable to me. I'll merge this, thanks again!

**Pull request merged**